core/amber/src/main/scala/edu/uci/ics/amber/engine/common/ambermessage/InvocationConvertUtils.scala

```diff
@@ -28,7 +32,20 @@ object InvocationConvertUtils {
 
   def returnInvocationToV1(
       returnInvocation: ReturnInvocationV2
-  ): ReturnInvocation =
-    ReturnInvocation(returnInvocation.originalCommandId, returnInvocation.controlReturn)
+  ): ReturnInvocation = {
+    println(controlReturnToV1(returnInvocation.controlReturn))
```

Comment:
is this a temporary print for debugging, or it's intentional logging?

Reply:
forgot to remove.